### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by tholu** [js-cookie/java-cookie#24]
After merging #21, #22, #23 - a new release would be nice. @FagnerMartinsBrack

**Comment by FagnerMartinsBrack** [comment]
Agree, I'm not going to release a 1.0.0, though. We need to fix the [NullPointer hell](https://github.com/js-cookie/java-cookie/pull/22#discussion_r261399385). Also, there's https://github.com/js-cookie/java-cookie/issues/12 and https://github.com/js-cookie/java-cookie/issues/9.

Once those are out of the way this should be good to be a v1 (or still stay on an alpha stage for a while to prove it's stable enough)

**Comment by tholu** [comment]
Sure, that suggestion was more tongue-in-cheek than serious 😉 Feel free to use whatever version number you might like.

**Comment by JevinMenezes** [comment]
v0.1.0 can be taken up as a maven release after the closure of #9 and #12

**Comment by JevinMenezes** [comment]
#27 , #29 and #30 are closed, out of them #27 was for the cookie attribute [SameSite](https://tools.ietf.org/id/draft-ietf-httpbis-rfc6265bis-03.html#the-samesite-attribute-1).
